From de002983174698762ecd3e42ab5a4b1bceb80af1 Mon Sep 17 00:00:00 2001
From: Stefan Wahren <stefan.wahren@i2se.com>
Date: Sun, 30 Dec 2018 12:07:16 +0100
Subject: [PATCH] ARM: dts: bcm2835-rpi: Drop unnecessary
 #address-cells/#size-cells

Compiling the bcm2835-rpi.dtsi with W=1 leads to the following warning:

Warning (avoid_unnecessary_addr_size): /soc/firmware: unnecessary
 #address-cells/#size-cells without "ranges" or child "reg" property

Fix this by removing these unnecessary properties.

Signed-off-by: Stefan Wahren <stefan.wahren@i2se.com>
Reviewed-by: Eric Anholt <eric@anholt.net>
---
 arch/arm/boot/dts/bcm2835-rpi.dtsi | 2 --
 1 file changed, 2 deletions(-)

--- a/arch/arm/boot/dts/bcm2835-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2835-rpi.dtsi
@@ -19,8 +19,6 @@
 	soc {
 		firmware: firmware {
 			compatible = "raspberrypi,bcm2835-firmware", "simple-bus";
-			#address-cells = <0>;
-			#size-cells = <0>;
 			mboxes = <&mailbox>;
 		};
 
